activesupport/lib/active_support/railtie.rb

```diff
@@ -47,5 +47,11 @@ class Railtie < Rails::Railtie # :nodoc:
         ActiveSupport.send(k, v) if ActiveSupport.respond_to? k
       end
     end
+
+    initializer "active_support.set_claims_expiration" do |app|
+      require 'active_support/claims'
+      app.config.active_support.claims_expiration ||= 1.month
```

Comment:
Well I think that ```Claims``` shouldn't be available to general public. It is just like a helper to ```MessageVerifier```. ```Claims``` is not much of use by itself.